### DISCUSSION TITLE
New Gateway Plugin doesnt show in storefront

### DISCUSSION THREAD

**Discussion by piotrosan** [saleor/saleor#9090] [General]
Hi, 

i'm new in saleor and i add new gateway plugin for payment. This gateway show in dashboard where i configure it but in shop, in storefront doesnt show, what are the criteria for a plugin to appear in the store ? I looked through the code but I thought I would ask first, maybe your hint will speed up my work. 

Thanks for help.
Piotr

**Comment by karolkielecki** [comment]
Hi @piotrosan, 

If you use the saleor-storefront repo, you can find info on how to add a payment gateway here: https://github.com/saleor/saleor-storefront#adding-a-payment-gateway
